### ISSUE TITLE
Warn about suspicious coalesce operation

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1706]
Hello!

This conditional is suspicious:

```php
if ($userId ?? null === 3) { /* ... */ }
```

Because it will acts like `$userId ?? (null === 3)` instead of `($userId ?? null) === 3`.

https://3v4l.org/5sh4K

**Comment by ktomk** [comment]
@rentalhost Do you have the Yoda setting on or off?

**Comment by ea-inspections-team** [comment]
Thanks for the proposal @rentalhost! The case is already reported by Php Inspestions (EA Ultimate).

Closing as a duplicate.
